pmm-app/src/shared/core/Settings.service.ts

```diff
@@ -3,8 +3,8 @@ import { API } from './constants';
 import { Settings, SettingsAPIResponse, SettingsPayload } from './types';
 
 export const SettingsService = {
-  async getSettings(disableNotifications = false): Promise<Settings> {
-    const { settings } = await apiRequest.post(API.SETTINGS, {}, disableNotifications) as SettingsAPIResponse;
+  async getSettings(): Promise<Settings> {
+    const { settings } = await apiRequest.get(API.SETTINGS) as SettingsAPIResponse;
```

Comment:
Please check the usage of settings as the api response changed (there are no longer enable/disable for each of the setting items only enable)

Reply:
Fixed.